include/aca_vlan_manager.h

```diff
@@ -34,15 +34,14 @@ namespace aca_vlan_manager
 struct vpc_table_entry {
   uint vlan_id;
 
-  uint32_t tunnel_id;
   // list of ovs_ports names on this host in the same VPC to share the same internal vlan_id
   list<string> ovs_ports;
+
   // hashtable of output (e.g. vxlan) tunnel ports to the neighbor host communication
-  // to neighbor port ID mapping in this VPC
-  // unordered_map <outports, list of neighbor port IDs>
+  // hashtable <key: outports string, value: list of neighbor port IDs>
   unordered_map<string, list<string> > outports_neighbors_table;
 
-  uint32_t oam_server_port;
+  uint oam_server_port;
```

Comment:
create issue to decouple gateway information

Reply:
Cool, modularization and decoupling is the way to go 👍